lib/srv/reexec.go

```diff
@@ -875,11 +876,11 @@ func newParker(ctx context.Context, credential syscall.Credential) error {
 // getCmdCredentials parses the uid, gid, and groups of the
 // given user into a credential object for a command to use.
 func getCmdCredential(localUser *user.User) (*syscall.Credential, error) {
-	uid, err := strconv.Atoi(localUser.Uid)
+	uid, err := apiutils.StrToUInt32(localUser.Uid)
```

Comment:
Is there any possible way that Go's `os/user` package would return a UID that's not valid? I would be surprised if that's the case, meaning this check is also not necessary.